CHMeetupApp/Sources/Common/Extensions/Foundation/String/String+Range.swift

```diff
@@ -10,12 +10,12 @@ import Foundation
 
 extension String {
   func substring(from index: Int) -> String {
-    return substring(from: self.index(startIndex, offsetBy: index))
+    return String(self[self.index(startIndex, offsetBy: index)...])
   }
 
   func substring(with range: Range<Int>) -> String {
-    let startIndex = self.index(self.startIndex, offsetBy: range.lowerBound)
-    let endIndex = self.index(self.startIndex, offsetBy: range.upperBound)
-    return substring(with: startIndex ..< endIndex)
+    let startIndex = index(self.startIndex, offsetBy: range.lowerBound)
+    let endIndex = index(self.startIndex, offsetBy: range.upperBound)
```

Comment:
let endInd = index(startIndex, offsetBy: range.upperBound)

Reply:
Here too.

Reply:
Noo, the answer above